app/helpers/metrics_helper.rb

```diff
@@ -12,51 +12,73 @@ def sum_rollups(rollups)
     rollups.sum(&:count)
   end
 
+  def podcast_main_card_options
+    %i[downloads episodes uniques dropdays]
+  end
+
+  def episode_main_card_options
+    %i[downloads]
+  end
+
   def interval_options
     Rollups::HourlyDownload::INTERVALS.map { |i| [I18n.t(".helpers.label.metrics.interval.#{i.downcase}"), i] }
   end
 
-  def date_range_options(episode = nil)
+  def date_preset_options(episode = nil)
```

Comment:
this is used in the `filter_dates` partial, which is reused between the Podcast Metrics and Episode Metrics views, which is why it's formatted with this conditional.